src/arrays/arraymath.jl

```diff
@@ -59,7 +59,7 @@ function *{B<:OrthonormalBasis}(qm1::QuMatrix{B}, qm2::QuMatrix{B})
                    bases(qm2,2))
 end
 
-*(dm1::DualMatrix, dm2::DualMatrix) = (dm1.qarr*dm2.qarr)'
+*(dm1::DualMatrix, dm2::DualMatrix) = QuArray(coeffs(dm1)*coeffs(dm2),bases(dm1,1), bases(dm2,2))
```

Comment:
Ah! I didn't even notice that I had switched them, really nice catch. We should have a test for that...